helm/alloy/values.yaml

```diff
@@ -10,15 +10,47 @@ networkPolicy:
     egress: []
     ingress: []
 
+psp:
+  enabled: true
+
 alloy:
   # You can make the whole chart ineffective by setting enabled to "false"
   enabled: true
 
+  alloy:
+    securityContext:
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+        - ALL
+      runAsUser: 473
+      runAsGroup: 473
+      runAsNonRoot: true
+      readOnlyRootFilesystem: true
+      seccompProfile:
+        type: RuntimeDefault
+
   image:
     repository: giantswarm/alloy
+
   configReloader:
     image:
       repository: giantswarm/configmap-reload
+    securityContext:
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+        - ALL
+      runAsUser: 473
+      runAsGroup: 473
+      runAsNonRoot: true
+      readOnlyRootFilesystem: true
+      seccompProfile:
+        type: RuntimeDefault
 
   serviceMonitor:
     enabled: true
+
+serviceAccount:
```

Comment:
```suggestion
  serviceAccount:
```

Reply:
I think that's why the check fails

Reply:
indentation is wrong, but this does not prevent check from failing

Reply:
sure

Reply:
removed as its no longer needed, and set by default in the upstream chart